src/presentation/components/Login/useLogin.ts

```diff
@@ -45,9 +47,14 @@ export const useLogin = () => {
     setSignupModalIsOpen(true);
   };
 
+  const loadUserData = async () => {
+    await getCreditCards();
+    await getBalance();
+  };
+
   useEffect(() => {
     if (isSuccess) {
-      getCreditCards();
+      loadUserData();
```

Comment:
As a note, this one will be moved to the isSuccess of our login, after the one with ftx.Otherwise, we wouldn't have our token to check the credit cards.